### PR TITLE
Add Approved queue to TSP

### PR DESCRIPTION
**Pull request by macrael**
## Description

Adds a new queue to the TSP site for approved shipments

## Code Review Verification Steps

* [x] End to end tests pass (`make e2e_test`).
* [x] There are no aXe warnings for UI.
* [ ] This works in IE.
* [ ] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/159670577) for this change

### CODE REVIEW

File: pkg/testdatagen/scenario/e2ebasic.go [hunk 1]
```diff
@@ -400,6 +400,13 @@ func (e e2eBasicScenario) Run(db *pop.Connection, loader *uploader.Uploader) {
 		log.Panic(err)
 	}
 
+	testdatagen.MakeServiceAgent(db, testdatagen.Assertions{
```
Comment (kilbergr) [inline pkg/testdatagen/scenario/e2ebasic.go]: What's this for? I don't think an AWARDED shipment needs one. But it's fine to leave in--just curious

File: pkg/testdatagen/scenario/e2ebasic.go [hunk 2]
```diff
@@ -527,7 +534,7 @@ func (e e2eBasicScenario) Run(db *pop.Connection, loader *uploader.Uploader) {
 	models.SaveMoveDependencies(db, &hhg5.Move)
 
 	/*
-	 * Service member with accepted shipment
+	 * Service member with in transit shipment
```
Comment (kilbergr) [inline pkg/testdatagen/scenario/e2ebasic.go]: Thank you 👼

File: pkg/testdatagen/scenario/e2ebasic.go (hunk 1 above)
Comment (macrael) [reply pkg/testdatagen/scenario/e2ebasic.go]: I figured there was no harm in having a more fully featured shipment, here.

**Comment by macrael** [comment]
@kilbergr comments addressed!

**Pull request merged**